cassandra/src/main/java/zipkin2/dependencies/cassandra/CassandraDependenciesJob.java

```diff
@@ -145,6 +151,7 @@ private static String getEnv(String key, String defaultValue) {
     if (builder.jars != null) conf.setJars(builder.jars);
     for (Map.Entry<String, String> entry : builder.sparkProperties.entrySet()) {
       conf.set(entry.getKey(), entry.getValue());
+      log.info("Spark conf properties: {}={}", entry.getKey(), entry.getValue());
```

Comment:
please change all these to debug level just in case there's a sensitive property.